src/components/organisms/AppRouter/AppRouter.tsx

```diff
@@ -93,10 +103,27 @@ const EmergencyViewPage = lazy(() =>
 export const AppRouter: React.FC = () => {
   const { user } = useUser();
 
+  // @debt custom redirects that are to be removed in the future
+  // done as per: https://github.com/sparkletown/internal-sparkle-issues/issues/1547
+  const googleCloudWestWorldUrl = generateAttendeeInsideUrl({
+    worldSlug: googleCloudWestName,
+    spaceSlug: googleCloudWestName,
```

Comment:
if you do `as WorldSlug` and `as SpaceSlug` on these and the ones below then you don't need to modify `utils/url.js`

Reply:
done 639cb5f